go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/stackrox/rox
 
-go 1.21
+go 1.21.8
```

Comment:
Sanity: This minor bump doesn't need to be aligned in the builder images used in CI, right?

Reply:
Unlikely since the builder images are newer than 1.21.8, but I'm not 100% sure about this

Reply:
I think it depends on the `GOTOOLCHAIN` setting in the builder image. This toolchain stuff is actually proving to be a pain

Reply:
As long as the builder image is bundled with a newer Go version, then it'll use that newer go version. See

https://go.dev/doc/toolchain
```
In the standard configuration, the go command uses its own bundled toolchain when that toolchain is at least as new as the go or toolchain lines in the main module or workspace. For example, when using the go command bundled with Go 1.21.3 in a main module that says go 1.21.0, the go command uses Go 1.21.3.
```